clipboard/android/src/main/java/com/capacitorjs/plugins/clipboard/Clipboard.java

```diff
@@ -0,0 +1,75 @@
+package com.capacitorjs.plugins.clipboard;
+
+import android.content.ClipData;
+import android.content.ClipDescription;
+import android.content.ClipboardManager;
+import android.content.Context;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
+
+public class Clipboard {
+
+    public enum ContentType {
+        STRING,
+        IMAGE,
+        URL
+    }
+
+    private Context context;
+    private String logTag = Logger.tags(this.getClass().getSimpleName());
+
+    public Clipboard(Context context) {
+        this.context = context;
+    }
+
+    public void write(String label, ContentType contentType, String content) {
+        ClipboardManager clipboard = (ClipboardManager) context.getSystemService(Context.CLIPBOARD_SERVICE);
```

Comment:
Is it normal to use `getSystemService()` several times in methods instead of once in the constructor?

Reply:
It's called twice in this class so its prob not a big deal but I did it and it looks cleaner so lets do it.